.github/workflows/ci.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
           - recover-id-object/Cargo.toml
           - generator/Cargo.toml
           - wallet-connect-test-bench/smart-contract/Cargo.toml
+          - notification-server/Cargo.toml
```

Comment:
The related issue will be closed automatically by this PR at merge when you add "closes #150" in the PR description which I recommend adding.